coordinator/src/main/resources/cluster-reference.conf

```diff
@@ -0,0 +1,244 @@
+######################################
+# Akka Cluster Reference Config File #
+######################################
+# NOTE: This is from Akka 2.3.15
+# It has to be named separately because the reference.conf conflicts with stuff in Spark classpath and
+# somehow doesn't get picked up.  :(
+
+# This is the reference config file that contains all the default settings.
+# Make your edits/overrides in your application.conf.
+
+akka {
+
+  cluster {
+    # Initial contact points of the cluster.
+    # The nodes to join automatically at startup.
+    # Comma separated full URIs defined by a string on the form of
+    # "akka.tcp://system@hostname:port"
+    # Leave as empty if the node is supposed to be joined manually.
+    seed-nodes = []
+
+    # how long to wait for one of the seed nodes to reply to initial join request
+    seed-node-timeout = 5s
+
+    # If a join request fails it will be retried after this period.
+    # Disable join retry by specifying "off".
+    retry-unsuccessful-join-after = 10s
+
+    # Should the 'leader' in the cluster be allowed to automatically mark
+    # unreachable nodes as DOWN after a configured time of unreachability?
+    # Using auto-down implies that two separate clusters will automatically be
+    # formed in case of network partition.
+    # Disable with "off" or specify a duration to enable auto-down.
+    auto-down-unreachable-after = off
+
+    # deprecated in 2.3, use 'auto-down-unreachable-after' instead
+    auto-down = off
+
+    # The roles of this member. List of strings, e.g. roles = ["A", "B"].
+    # The roles are part of the membership information and can be used by
+    # routers or other services to distribute work to certain member types,
+    # e.g. front-end and back-end nodes.
+    roles = []
+
+    role {
+      # Minimum required number of members of a certain role before the leader
+      # changes member status of 'Joining' members to 'Up'. Typically used together
+      # with 'Cluster.registerOnMemberUp' to defer some action, such as starting
+      # actors, until the cluster has reached a certain size.
+      # E.g. to require 2 nodes with role 'frontend' and 3 nodes with role 'backend':
+      #   frontend.min-nr-of-members = 2
+      #   backend.min-nr-of-members = 3
+      #<role-name>.min-nr-of-members = 1
+    }
+
+    # Minimum required number of members before the leader changes member status
+    # of 'Joining' members to 'Up'. Typically used together with
+    # 'Cluster.registerOnMemberUp' to defer some action, such as starting actors,
+    # until the cluster has reached a certain size.
+    min-nr-of-members = 1
+
+    # Enable/disable info level logging of cluster events
+    log-info = on
+
+    # Enable or disable JMX MBeans for management of the cluster
+    jmx.enabled = on
```

Comment:
Sorry, this config file was copied from Akka itself, because for some reason reference.conf gets overridden by Spark class path and the cluster settings are lost.  :/